app/code/community/Bolt/Boltpay/Helper/Data.php

```diff
@@ -177,7 +177,11 @@ public function buildOnSuccessCallback($successCustom, $checkoutType)
      */
     public function buildOnCloseCallback($closeCustom, $checkoutType)
     {
-        $successUrl = $this->getMagentoUrl(Mage::getStoreConfig('payment/boltpay/successpage'));
+        // For frontend URLs, we want to session id process the URL to get the correct URL
```

Comment:
Can you clarify this comment a little? I know what you're doing but the comment doesn't make sense. Good work.

Reply:
@johnmccombs1 
Would it make more sense with quotations like this:
> For frontend URLs, we want to _"session id process"_ the URL to get the final format URL which may or may not contain the `__SID=SU`

Reply:
Also, if I edit the comment, it will have to re-reviewed.  So, the comment will come in a separate PR

Reply:
@mccottry I think it makes more sense to just edit the comment in this PR. It results in the same number of reviews, but 1 less PR and less confusion